spec/pageflow/policies/entry_policy_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'spec_helper'
+
+module Pageflow
+  module Policies
+    describe EntryPolicy do
+      terms = {class: EntryPolicy,
+               subject: FactoryGirl.create(:entry),
+               name: 'entry'}
+      describe :configure do
+        include_examples 'for entry and account',
+                         terms: terms,
+                         permission_type: :configure,
+                         minimum_required_role: 'manager',
+                         maximum_forbidden_role: 'publisher'
+      end
+
+      describe :publish do
+        include_examples 'for entry and account',
+                         terms: terms,
+                         permission_type: :publish,
+                         minimum_required_role: 'publisher',
+                         maximum_forbidden_role: 'editor'
+      end
+
+      describe :edit do
+        include_examples 'for entry and account',
+                         terms: terms,
+                         permission_type: :edit,
+                         minimum_required_role: 'editor',
+                         maximum_forbidden_role: 'previewer'
+      end
+
+      describe :preview do
+        include_examples 'for entry and account',
```

Comment:
this shared example name is too global. If I understand `include_examples` correctly the description is not visible in the resulting spec description, right? So we should b free to choose a more descriptive name.